buildpacks/conda-nodefaults/bin/build

```
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+echo -e "🔧 \033[1mConda Channel Cleanup Buildpack\033[0m"
+
+ENV_FILE="environment.yml"
+
+# Exit silently if environment.yml is not present
+[[ -f "$ENV_FILE" ]] || exit 0
```

suggestion: It is nicer to log what happened and exit. Rather than doing it fully silently.

```suggestion
if [ ! -f "$ENV_FILE" ]; then
  echo "Cannot find $ENF_FILE, exiting silently."
  exit 0
fi
```